atphy/masksnowtn
# Cb vuetify

#### Changes Made
1. added `node_modules` to .gitignore. was in file but was not ignoring. necessary for this PR because installing vuetify dependency added `node_modules` file
2. added vuetify dependency to project. not confident in implementation so would encourage a double check

#### Steps to Review
1. Checkout this branch locally and run the application.
2. click on the commits pertaining to vuetify to check if it was done properly. the files that ignore 
```
git fetch 
git checkout cb-vuetify
```
2. Run the application.
```
serve
```

3. Test app functionality.
> node_modules should be ignored
> developer should now be able to use vuetify
4. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
